x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/examples_list/examples_list.tsx

```diff
@@ -39,7 +44,9 @@ export const ExamplesList: FC<Props> = ({ examples }) => {
         </h6>
       </EuiText>
       <EuiSpacer size="s" />
-      {examplesContent}
+      <EuiListGroup flush={true} showToolTips={true}>
```

Comment:
The change to the `EuiListGroup` here is giving the content left alignment. I think that works well for `text` cards where the examples general use more than full width:

![image](https://user-images.githubusercontent.com/7405507/73459731-44a02c00-436f-11ea-8074-d8a71bbcc375.png)

but not so sure it works for `geo` fields as well:

![image](https://user-images.githubusercontent.com/7405507/73459867-7b764200-436f-11ea-923c-834106066739.png)

How does it look if you give it `center` alignment?

Reply:
Fixed in https://github.com/elastic/kibana/pull/56372/commits/606e182d5c59d068e9ddda37ee98938fe607b776